README.md

```diff
@@ -1,5 +1,7 @@
 # The OpenTracing Specification repository
 
+OpenTracing Specification Official Website: [http://opentracing.io/spec](http://opentracing.io/spec).
```

Comment:
@wu-sheng this is not the endgame I'd like here... let's close this and I will open an issue to resolve this properly from the other side. https://github.com/opentracing/opentracing.io/issues/177